app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -850,8 +850,8 @@ def test_reject_ending_experiment(self):
 
     def test_end_experiment(self):
         user_email = "user@example.com"
-        experiment = NimbusExperimentFactory.create(
-            status=NimbusExperiment.Status.LIVE,
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            lifecycle=NimbusExperimentFactory.Lifecycles.PUBLISH_APPROVE_APPROVE
```

Comment:
just a suggestion: this name is little confusing `PUBLISH_APPROVE_APPROVE` 😆

Reply:
I agree that it's a bit confusing, especially when there are so many lifecycle states

The thought in changing these names from `LIVE_` to `PUBLISH_` was to match the names to the sequence diagram names: https://github.com/mozilla/experimenter/blob/main/app/experimenter/docs/README.md#publish-approveapprove

Reply:
If you think it's too confusing I can change them back though